etc/profile-a-l/ani-cli.profile

```diff
@@ -0,0 +1,42 @@
+# Firejail profile for ani-cli
+# Description: Shell script to watch Anime from the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include ani-cli.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+noblacklist ${HOME}/.cache/ani-cli
+noblacklist ${HOME}/.local/state/ani-cli
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+include disable-proc.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/ani-cli
+mkdir ${HOME}/.local/state/ani-cli
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.cache/ani-cli
+whitelist ${HOME}/.local/state/ani-cli
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+
+#machine-id
+nodvd
+noprinters
+notv
+
+disable-mnt
+private-bin ani-cli,aria2c,cat,cp,curl,cut,ffmpeg,fzf,grep,head,mkdir,mv,nl,sed,sh,sort,tput,tr,uname,wc
+#private-cache
+private-etc alsa,alternatives,asound.conf,ca-certificates,crypto-policies,fonts,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,machine-id,mime.types,nsswitch.conf,pango,pki,protocols,pulse,resolv.conf,rpc,services,ssl,X11,xdg
+private-tmp
+
+read-only ${HOME}/.config/mpv
+
+# Redirect
+include mpv.profile
```

Comment:
mpv is just the default. Other player like vls are supported to. Redirecting it to mpv makes it harder for users to create locals that allow other players.

Reply:
I can understand the skepticism. It's a bit similar to Firejail's favoritism towards Firefox, which is very understandable from the history of the project. Think fireurl (regardless of implementation) with multi web-browser functionality. On occasion I've been playing with the idea to write such a meta-web-browser.profile that would tackle these kind of 'issues'. Nothing that's very concrete yet though. In a similar vain we might consider a meta-media-player.profile that supports whatever media-player applications Firejail currently supports.

Totally out of scope for the currently opened PR's from @pirate486743186 IMO. But something to have a wider discussion on at some point?

Reply:
@pirate486743186 @rusty-snake Any thoughts on #5712 to unbreak mpv lock-in?

Reply:
@glitsj16 [on Mar 7](https://github.com/netblue30/firejail/pull/5707#discussion_r1127767622):

> Any thoughts on #5712 to unbreak mpv lock-in?

I have something half-written up; I'll post it in #5712 later.

Reply:
@kmk3 No rush. I just wanted to avoid the topic getting snowed under. Looking forward to reading your views!